okcupyd/html_search.py

```diff
@@ -218,8 +218,7 @@ def SearchFetchable(session=None, **kwargs):
     :param location: A location string which will be used to filter results.
     :param gender: The gender of the user performing the search.
     :param keywords: A list or space delimeted string of words to search for.
-    :param order_by: The criteria to use for ordering results. expected_values:
-                     'match', 'online', 'special_blend'
+    :param order_by: Expected values: 'match', 'online', 'special_blend'
```

Comment:
No good reason. At one point it seemed that the docstrings hardly ever provided both a `description` and a `accepted_values`, and I guess that's what prompted this. Later though I saw that some locations did use both.